pkgs/development/guile-modules/guile-git/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchFromGitHub, fetchFromGitLab, guile, libgit2
+, autoreconfHook, pkg-config, texinfo
+}:
+
+let
+  bytestructures = stdenv.mkDerivation rec {
+    pname = "scheme-bytestructures";
+    version = "1.0.7";
+
+    src = fetchFromGitHub {
+      owner = "TaylanUB";
+      repo = pname;
+      rev = "v${version}";
+      sha256 = "0q0habjiy3h9cigb7q1br9kz6z212dn2ab31f6dgd3rrmsfn5rvb";
+    };
+
+    postConfigure = ''
+      sed -i '/moddir\s*=/s%=.*%=''${out}/share/guile/site%' Makefile;
+      sed -i '/godir\s*=/s%=.*%=''${out}/share/guile/ccache%' Makefile;
```

Comment:
Same comment from before applies here. If we need to do this multiple times, that's even more reason to find a better way.

Reply:
I'm open to ideas, but it's hard coded in the Makefile. I don't see any other way beyond text manipulation